README.md

```diff
@@ -1,85 +1,60 @@
+1. The `[mail]` section configures the SMTP server. Matomo uses mailers for several core functions, including password resets and reporting. Some related config values (e.g., `login_password_recovery_email_address`) are defined in the `[General]` section.
+1. The `[General]` section handles other configurations. A few of these settings are covered by the EnvironmentVariables plugin, but the rest are managed in this code.
+
+By using the [EnvironmentVariables](https://plugins.matomo.org/EnvironmentVariables) plugin for Matomo, we generally don't need to store much in the `config.ini.php` file. Instead, the [mitlib-tf-workloads-matomo](https://github.com/MITLibraries/mitlib-tf-workloads-matomo) code defines the key environment variables for the ECS service, overriding anything in `config.ini.php` and `global.ini.php`.
+
+The database stores the rest of the state of Matomo: all the data itself along with the Superuser credentials and the other user accounts and credentials.
 
 ### Data anonymization
 
-We anonymize tracking data to protect our users' privacy. Specifically, we mask
-two bytes of visitors' IPs (e.g., 192.186.x.x), and we have disabled [Matomo's
-User ID plugin](https://matomo.org/docs/user-id/).
+We anonymize tracking data to protect our users' privacy. Specifically, we mask two bytes of visitors' IPs (e.g., 192.186.x.x), and we have disabled [Matomo's User ID plugin](https://matomo.org/docs/user-id/).
 
-Before promoting a new build to production, ensure that IPs are anonymized by
-visiting Administration -> Privacy -> Anonymize data in the GUI. (You shouldn't
-ever need to modify these settings, but it's important to verify as part of the
-deploy process.)
+Before promoting a new build to production, ensure that IPs are anonymized by visiting Administration -> Privacy -> Anonymize data in the GUI. (You shouldn't ever need to modify these settings, but it's important to verify as part of the deploy process.)
 
 ### Archiving reports
 
-Matomo calls the process by which it compiles raw log data into human-readable
-report 'archiving'. By default, archiving occurs on demand, whenever a Matomo
-user attempts to view a report in the GUI. Based on [Matomo's recommendation](https://matomo.org/docs/setup-auto-archiving/), we have configured a cron job to archive reports every
-hour.
-
-The cron job runs as a scheduled ECS task and is defined in the
-[Terraform config](https://github.com/MITLibraries/mitlib-terraform/tree/master/apps/analytics).
+Matomo calls the process by which it compiles raw log data into human-readable report 'archiving'. By default, archiving occurs on demand, whenever a Matomo user attempts to view a report in the GUI. While [Matomo's recommendation](https://matomo.org/docs/setup-auto-archiving/) is to schedule this, we have **not** configured a cron job to archive reports at this time.
```

Comment:
As this is a change from the current production, we should check in with UX to confirm data is being made available to reports in a timely manner (i.e. do reports end up with correct data with this on-demand process and just run a bit slower, or does the archive process kick off but the report finishes quickly but with outdated data, etc)

Reply:
oops. I did in fact add back the hourly report archiving (this is built in the [mitlib-tf-workloads-matomo](https://github.com/MITLibraries/mitlib-tf-workloads-matomo/blob/dev/ecs-report-archive-task.tf) repo, but I forgot to update this note in the README here!